LiipImagineBundle.php

```diff
@@ -32,7 +33,11 @@ public function build(ContainerBuilder $container)
         $extension = $container->getExtension('liip_imagine');
 
         $extension->addResolverFactory(new WebPathResolverFactory());
-        $extension->addResolverFactory(new AwsS3ResolverFactory());
+        if (defined('\Aws\Sdk::VERSION') && version_compare(\Aws\Sdk::VERSION, '3.0.0', '>=')) {
```

Comment:
register both, and let a developer decided which one to use. Also would be good to add an exception if a resolver is used with wrong sdk.